README.md

```diff
@@ -1,2 +1,17 @@
 # sourcerer-app
 Sourcerer app makes engineering profiles from their git repositories.
+
+Install/uninstall
+=================
+
+Run the command to install `sourcerer` tool on your machine:
+
+```
+curl -s https://sourcerer.io/install | sudo bash
+```
+
+Run the command to remove it from your machine:
+
+```
+curl -s https://sourcerer.io/uninstall | sudo bash
```

Comment:
Uninstallation is not just a jar. There is also a config file, there can be some sort of caching files. Maybe temp files. Formats of those will change, we need to be careful. So yes, I would rather have an uninstall script locally. In Unix, a program can remove itself, but in Windows last time I checked it was not the case. I would prefer a separate script.